en/guides/user-management/README.md

```diff
@@ -0,0 +1,127 @@
+---
+title: User management
+description: "Tutorial: user management"
+---
+
+# User management
+
+All **users** can [sign in](/en/for-students#sign-in) and manage their [user profile](/en/for-students#user-profile). Some actions on Dodona require more [permissions](#permissions). For this reason, we use three roles: [students](#student), [teachers](#teacher) and [admins](#admin). These roles can be [assigned](#assign-roles) to users, for instance a teacher can appoint another teacher.
+
+::: tip
+
+You can see that you are a [teacher](#teacher) when you can see the **management menu** (menu <span class="guilabel">Admin</span>) left of the [user menu](/en/for-students#user-menu) at the right side of the [navigation drawer](/en/for-students#navigation-drawer).
```

Comment:
The management menu is now incorporated into the drawer:
![HkpR](https://user-images.githubusercontent.com/3226995/90104672-f835a000-dd44-11ea-841d-39a31a6f122b.png)